lib/milc_interface.cpp

```diff
@@ -1136,7 +1136,7 @@ void qudaEigCGInvert(int external_precision,
 {
   static const QudaVerbosity verbosity = QUDA_VERBOSE; //getVerbosity();
   qudamilc_called<true>(__func__, verbosity);
-
+#if 0
```

Comment:
Does the MILC interface need further updates?  If not, presumably we should delete the `#if 0` statements.